app/views/blocks/imageset-demo.html

```diff
@@ -7,7 +7,7 @@ <h2 class="demo__title">Image set</h2>
 		<div class="imageset-demo">
 			{% for image in imageset_list %}
 				<div class="imageset-demo__item">
-					<img src="https://www.ft.com/__origami/service/image/v2/images/raw/{{imageset_scheme}}:{{image.name}}?source=origami-registry" alt="{{image.name}}" class="imageset-demo__image">
+					<img src="https://www.ft.com/__origami/service/image/v2/images/raw/{{imageset_scheme}}:{{image.name}}%3Fcache-bust%3D{{"now"|date("U")}}?source=origami-registry" alt="{{image.name}}" class="imageset-demo__image">
```

Comment:
Maybe this should use 10-minute intervals or something? So Cloudinary doesn't pull in millions of duplicate images.